Wellthier/View Controllers/WorkoutViewController.h

```diff
@@ -0,0 +1,26 @@
+//
+//  WorkoutViewController.h
+//  Wellthier
+//
+//  Created by Craig Lee on 7/14/22.
+//
+
+#import "ViewController.h"
+#import "Workout.h"
+
+@import Parse;
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface WorkoutViewController : ViewController
+
+@property (weak, nonatomic) IBOutlet UILabel *workoutTitle;
+@property (weak, nonatomic) IBOutlet UILabel *author;
+@property (strong, nonatomic) NSArray *arrayOfExercises;
+@property (strong, nonatomic) NSArray *filteredExercises;
+@property (strong, nonatomic) Workout *detailWorkout;
+@property (weak, nonatomic) IBOutlet PFImageView *workoutImageView;
```

Comment:
I would like to keep these properties in consistent order for ex:
(nonatomic, weak)
(nonatomic, strong)